plutus-benchmark/marlowe/src/PlutusBenchmark/Marlowe/Scripts/Semantics.hs

```diff
@@ -1,55 +1,37 @@
-
 -- editorconfig-checker-disable-file
-
-
------------------------------------------------------------------------------
---
--- Module      :  $Headers
--- License     :  Apache 2.0
---
--- Stability   :  Experimental
--- Portability :  Portable
---
--- | Marlowe validators.
---
------------------------------------------------------------------------------
-
-
 {-# LANGUAGE DataKinds             #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE NamedFieldPuns        #-}
 {-# LANGUAGE OverloadedStrings     #-}
 {-# LANGUAGE RecordWildCards       #-}
 {-# LANGUAGE TemplateHaskell       #-}
 {-# LANGUAGE ViewPatterns          #-}
-
 {-# OPTIONS_GHC -Wno-name-shadowing #-}
-
-{-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:defer-errors #-}
 {-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:context-level=0 #-}
+{-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:defer-errors #-}
 {-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:target-version=1.0.0 #-}
 
-module PlutusBenchmark.Marlowe.Scripts.Semantics
-  ( -- * Types
-    MarloweInput
-  , MarloweTxInput(..)
-    -- * Semantics Validator
-  , marloweValidatorHash
-  , marloweValidatorBytes
-  , marloweValidator
-  , mkMarloweValidator
-    -- * Utilities
-  , marloweTxInputsFromInputs
-  ) where
+module PlutusBenchmark.Marlowe.Scripts.Semantics (
+  -- * Types
+  MarloweInput,
+  MarloweTxInput (..),
+
+  -- * Semantics Validator
+  marloweValidatorHash,
+  marloweValidatorBytes,
+  marloweValidator,
+  mkMarloweValidator,
 
+  -- * Utilities
+  marloweTxInputsFromInputs,
+) where
 
 import GHC.Generics (Generic)
 import PlutusBenchmark.Marlowe.Core.V1.Semantics as Semantics (MarloweData (..),
                                                                MarloweParams (MarloweParams, rolesCurrency),
-                                                               Payment (..),
-                                                               TransactionError (TEAmbiguousTimeIntervalError, TEApplyNoMatchError, TEHashMismatch, TEIntervalError, TEUselessTransaction),
+                                                               Payment (..), TransactionError (..),
```

Comment:
Why did you make this change? Did fourmolu do it or was it you? 

Also, please please please make it explicit that you also made other changes in this PR, since the title and description were misleading! 

Ideally, do not combine reformatting with refactoring. If this was a purely formatting PR I would be able to review it quickly. Given that it isn't, I keep asking you why some changes appear and it's not a good use of any of our time.